mappings/net/minecraft/registry/Registerable.mapping

```diff
@@ -0,0 +1,9 @@
+CLASS net/minecraft/unmapped/C_hqoyyfco net/minecraft/registry/Registerable
```

Comment:
I'm pretty sure this class used to be BootstrapContext, why the change?

Reply:
I don't remember changing this, but I can't find any trace of either a `BootstrapContext` or `Registerable` class in the past few branches. I'll change this to BootstrapContext.

Reply:
works for me! I'll look into it a little more when I have access to enigma